open-source-labs/reactime
# fix D3 NaN error on sites w no render data

## Description:
fix D3 NaN error on sites w no render data

## Changes I Made:
PerfView.tsx added error checking for missing data before zooming + ESLint auto fixes

## How to Test:
Run on unsupported site (Codesmith.io, which uses redux for example) and switch to Performance tab. Make sure no errors appear in Chrome background console.
